src/components/DetailCard/DetailCard.tsx

```diff
@@ -34,10 +38,8 @@ DetailCardProps) {
         <img src={Star} />
         {vote_average}
       </p>
-      {/* <p>{genres}</p> */}
-      <button className={styles.button}>
-        Add to Watchlist
-        <img src={Add} />
+      <button className={styles.heart} onClick={handleClick}>
+        <img src={Heart} />
```

Comment:
One of the coaches recommended me to turn svg images into components. Maybe you want to do the same?